scripts/minimal_init.lua

```diff
@@ -8,7 +8,7 @@ vim.filetype.add {
     cmm = "t32",
     hurl = "hurl",
     ncl = "nickel",
-    textproto = "textproto",
+    textproto = "pbtxt",
```

Comment:
```suggestion
```
If `pbtxt` is _the_ filetype, then this is not needed.

Reply:
The test files are using `*.textproto` as their extension.

Reply:
I thought #5402 had added test files using the `*.textproto`  extension, I was mistaken.

Reply:
Even if so, these could be changed ;)